app/controllers/users_controller.rb

```diff
@@ -1,13 +1,15 @@
+# frozen_string_literal: true
+
 class UsersController < ApplicationController
   def new
-    @user ||= User.new
+    @user = User.new
   end
 
   def create
     @user = User.new(user_params)
 
     if @user.save
-      redirect_to new_user_path
+      redirect_to root_path
```

Comment:
This should have redirected to `new_user_path` as you had it previously, please see the requirement [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) [`REQUIRED`]

Reply:
Thanks, I'll be more careful next time.